setup.py

```diff
@@ -7,25 +7,14 @@
 with open('mediameter/__init__.py', 'r') as fd:
     version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE).group(1)
 
-if sys.argv[1]=="sdist":
```

Comment:
Oops, this change is a little abrupt, so maybe reject this particular commit. OTOH, consider that pandoc is a heavy requirement for installing from source.